lib/event_tracer/dynamo_db_log_worker.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+module EventTracer
+  class DynamoDBLogWorker
+    include ::Sidekiq::Worker
+
+    sidekiq_options retry: 1, queue: 'low'
+
+    TABLE_NAME = ENV.fetch('AWS_DYNAMODB_LOGGING_TABLE', 'logs')
+    # See https://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/DynamoDB/Client.html#batch_write_item-instance_method
+    MAX_DYNAMO_DB_ITEM_PER_REQUEST = 25
+
+    def perform(items)
+      wrap(items).each_slice(MAX_DYNAMO_DB_ITEM_PER_REQUEST) do |batch|
+        data = batch.map do |item|
+          { put_request: { item: clean_empty_values(item) } }
+        end
+
+        DynamoDBClient.call.batch_write_item(
+          request_items: { TABLE_NAME => data }
+        )
+
+      rescue Aws::DynamoDB::Errors::ServiceError => e
```

Comment:
Could we also require `aws-dynamodb` and raise error if it's not loaded?

Reply:
Okay!